File: spec/php/ImportsAbsAbsTest.php
<?php
namespace Kaitai\Struct\Tests;

class ImportsAbsAbsTest extends TestCase {
    public function testImportsAbsAbs() {
        $r = ImportsAbsAbs::fromFile(self::SRC_DIR_PATH . '/fixed_struct.bin');

        $this->assertSame(80, $r->one());
        $this->assertSame(65, $r->two()->one());
        $this->assertSame(67, $r->two()->two()->one());
    }
}

File: spec/go/spec/imports_abs_abs_test.go
package spec

import (
	"runtime/debug"
	"os"
	"testing"
	"github.com/kaitai-io/kaitai_struct_go_runtime/kaitai"
	. "test_formats"
	"github.com/stretchr/testify/assert"
)

func TestImportsAbsAbs(t *testing.T) {
	defer func() {
		if r := recover(); r != nil {
			debug.PrintStack()
			t.Fatal("unexpected panic:", r)
		}
	}()
	f, err := os.Open("../../src/fixed_struct.bin")
	if err != nil {
		t.Fatal(err)
	}
	s := kaitai.NewStream(f)
	var r ImportsAbsAbs
	err = r.Read(s, &r, &r)
	if err != nil {
		t.Fatal(err)
	}

	assert.EqualValues(t, 80, r.One)
	assert.EqualValues(t, 65, r.Two.One)
	assert.EqualValues(t, 67, r.Two.Two.One)
}
